src/content/en/tools/workbox/v3/reference-docs/latest/workbox-cache-expiration_CacheExpirationPlugin.mjs.html

```diff
@@ -0,0 +1,239 @@
+<!DOCTYPE html>
```

Comment:
Can you get the auto-generation to insert the alpha warning (`.../workbox/v3/_shared/alpha.html`) at the top of each of these reference docs?

Reply:
No sorry - this is using the DevSite friendly theme and I wouldn't crazy about altering that as it might break.